scripts/fuck.bat

```diff
@@ -0,0 +1,3 @@
+@set PYTHONIOENCODING=utf-8
+@powershell -noprofile -c "cmd /c \"$(thefuck $(doskey /history)[-2])\""
```

Comment:
If I'm not missing something, I reckon arguments such as `--yeah` and `-r` ([reference](https://github.com/nvbn/thefuck/#installation)) are not relayed to `thefuck`. Would that be possible?

Reply:
%* did the trick here